agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulse.java

```diff
@@ -46,7 +46,8 @@ public static QuickPulse create(
       Supplier<String> instrumentationKey,
       @Nullable String roleName,
       @Nullable String roleInstance,
-      boolean useNormalizedValueForNonNormalizedCpuPercentage) {
+      boolean useNormalizedValueForNonNormalizedCpuPercentage,
+      @Nullable String sdkVersion) {
```

Comment:
do we ever pass `null` here? (wondering if we need the `@Nullable`)

Reply:
I passed null to [LiveMetricsSpanProcessor]([LiveMetricsSpanProcessor.java](https://github.com/microsoft/ApplicationInsights-Java/pull/2296/commits/3ba0168a4df9f7f03636060adeb36a5318424965#diff-6dca72a20c8b72cb57639d61cd18c481777a894721416fdab81be6a6db6bce58)). Updated it to take sdkVersion as an input parameter now.